PyInstaller/building/utils.py

```diff
@@ -239,7 +239,10 @@ def checkCache(fnm, strip=False, upx=False, dist_nm=None):
 
     if not os.path.exists(os.path.dirname(cachedfile)):
         os.makedirs(os.path.dirname(cachedfile))
-    shutil.copy2(fnm, cachedfile)
+    try:
+        shutil.copy2(fnm, cachedfile)
+    except:
```

Comment:
Please restrict this to a more concrete exception type. Also please add a comment why falling back to `copy()` and why this then is expected to work.